mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length;
 }
 
-function multiply(a, b, c) {
-  a * b * c;
-  return;
+function multiply(a,b,c) {
+  let times= a*b*c;
+  return times ;
 }
```

Comment:
Consider: This is how I wrote. Just different names. 
function multiply(a, b, c) {
  const answer = a * b * c;
  return answer;
}

Reply:
@softacoder have you used this button in the Github UI?
<img width="317" alt="image" src="https://user-images.githubusercontent.com/25401570/221957506-074360c2-8fdd-4393-a7e9-f96ae10d7070.png">